app/src/main/res/layout/topic_revision_summary_view.xml

```diff
@@ -22,7 +22,7 @@
     android:layout_marginBottom="@dimen/topic_revision_summary_view_container_margin_bottom"
     android:clipToPadding="true"
     android:onClick="@{(v) -> viewModel.onRevisionItemPressed.onTopicRevisionSummaryClicked(viewModel.subtopic)}"
-    app:cardBackgroundColor="@color/color_def_white"
+    app:cardBackgroundColor="@color/component_color_shared_screen_primary_background_color"
```

Comment:
Update this, similar to the comment left for `RevisionCardActivity` files.

Reply:
Done.